internal/services/containerapps/helpers/container_apps.go

```diff
@@ -1456,6 +1457,12 @@ func ContainerVolumeSchema() *pluginsdk.Schema {
 					ValidateFunc: validate.ManagedEnvironmentStorageName,
 					Description:  "The name of the `AzureFile` storage. Required when `storage_type` is `AzureFile`",
 				},
+
+				"mount_options": {
+					Type:        pluginsdk.TypeString,
+					Required:    false,
```

Comment:
`Required` is false by default, we should be setting `Optional` to `true` here
```suggestion
					Optional:    true,
```